### ISSUE TITLE
Create Explicit Allow - Deny all others - seccomp profile

### ISSUE THREAD

**Issue by jimmyraywv** [shamedgh/confine#5]
Given the concept of least privilege, have you thought about setting a mode in the process that creates a profile where the default action is `SCMP_ACT_ERRNO` and then explicitly list syscalls for the `SCMP_ACT_ALLOW` action?

**Comment by shamedgh** [comment]
Yes! We will add an option to create an allow list instead of the current deny list.
